micrometer-registry-newrelic-telemetry/runtime/src/main/java/io/quarkiverse/micrometer/registry/newrelic/telemetry/NewRelicConfig.java

```diff
@@ -0,0 +1,89 @@
+package io.quarkiverse.micrometer.registry.newrelic.telemetry;
+
+import java.util.Map;
+import java.util.Optional;
+
+import io.quarkus.micrometer.runtime.config.MicrometerConfig;
+import io.quarkus.runtime.annotations.ConfigItem;
+import io.quarkus.runtime.annotations.ConfigPhase;
+import io.quarkus.runtime.annotations.ConfigRoot;
+
+public class NewRelicConfig {
+
+    @ConfigRoot(name = "micrometer.export.newrelic.telemetry", phase = ConfigPhase.BUILD_AND_RUN_TIME_FIXED)
```

Comment:
This adjusted config root doesn't match the prefix (see below).

Reply:
fixed